ios-release-build.md

```diff
@@ -0,0 +1,198 @@
+---
+layout: page
+title: Preparing an iOS App for Release
+sidebar: home_sidebar
+permalink: /iOS-release/
+---
+
+During the typical development cycle, you'll test an app using `flutter run` at
+the command line, the Run and Debug toolbar buttons in IntelliJ, or both. By
+default, Flutter builds a *debug* version of your app.
+
+When you're ready to prepare a *release* version, for example to [publish to
+the App Store][appstore], follow the steps on this page. To release an app to
+the App Store, you will need a registered [Apple Developer Program][devprogram]
+account.
+
+This guide provides a step-by-step walkthrough of releasing a typical app to
+the App Store. For a detailed overview, see Apple's [App Distribution
+Guide][distributionguide].
+
+*Note*: Many of the steps below must be carried out in Xcode. To open your app
+in Xcode:
+
+  * From Xcode: Invoke `File > Open` and select the `ios` folder located inside
+    your main app directory.
+
+  * From the terminal: `cd <app dir>; open ios/Runner.xcworkspace`.
+
+* TOC Placeholder
+{:toc}
+
+
+## Register your app on iTunes Connect
+
+Before uploading a build to the App Store, you'll need to register your app on
+[iTunes Connect][itunesconnect]. iTunes Connect is where you'll manage your
+application's metadata, pricing and releases to the App Store.
+
+### Register a Bundle ID for your app
+
+Every iOS application is associated with a Bundle ID, which is a unique
+identifier registered with Apple. To register a bundle identifier for your app,
+follow these steps:
+
+1. Open the [App IDs][devportal_appids] page of your developer account.
+
+1. Click '+' to create a new Bundle ID.
+
+1. Enter an app name and select `Explicit App ID` and enter an ID.
+
+1. Select any services, then click `Continue`.
+
+1. On the next page, confirm the details and click `Register` to register your
+   Bundle ID.
+
+### Create an iTunes Connect application record
+
+The next step is to register your app on iTunes Connect. Follow these steps:
```

Comment:
Covered above (ll. 36-37) in the `##` heading for the overall iTunes Connect step, under which the bundle ID and this subsection are nested.